README.md

```diff
@@ -15,9 +21,13 @@ different collections in Firestore too. Initialise the class
 by passing a collection reference created by the Firestore library.
 
 ```typescript
-export interface UserDoc {
-  // a string id value is required for entities using this library.
-  // It will be used for the firestore document ID but not stored in the document in firestore.
+import { FirestoreDataSource } from 'apollo-datasource-firestore';
+
+export interface UsersDoc {
```

Comment:
```suggestion
export interface UserDoc {
```
I prefer the Doc types to be singular, looks like I didn't do that for Posts though, you have a collection called users that contains many user documents